packages/teleport/src/Kubes/Kubes.tsx

```diff
@@ -67,14 +70,8 @@ export function Kubes(props: State) {
       <FeatureHeader alignItems="center" justifyContent="space-between">
         <FeatureHeaderTitle>Kubernetes</FeatureHeaderTitle>
         {!hasNoKubes && (
-          <ButtonPrimary
-            as="a"
-            width="240px"
-            target="_blank"
-            href={DOC_URL}
-            rel="noreferrer"
-          >
-            View documentation
+          <ButtonPrimary width="240px" onClick={() => setShowAddKube(true)}>
+            Add Kubernetes
```

Comment:
i think we need to add a `ButtonAdd` equivalent that checks if user has perm to create a token (like we do in apps, nodes, etc), maybe even bring it out to component/ level to share it since the only thing that seems to differ is the resource name: `ResourceButtonAdd`? or something like that

Reply:
I'm glad you noticed that, thanks!

I've added the component in this commit https://github.com/gravitational/webapps/pull/750/commits/ba34e77a26f2d50db108b71cab20330a1da53cb3

And replaced the old buttons form apps and nodes in this one https://github.com/gravitational/webapps/pull/750/commits/f8e98516e191b7894cda9c2425320c52420809fc (Note that the snapshots have changed casing of the resource name but the CSS of the button changes all text to uppercase anyway)

Please let me know your thoughts